client/src/constants/participantTableConstants.js

```diff
@@ -108,11 +108,15 @@ export const FILTERABLE_FIELDS = {
   LIVED_LIVING_EXPERIENCE: 'livedLivingExperienceFilter',
 };
 
-const HCA = 'HCA';
-const MHAW = 'MHAW';
-export const allPrograms = [HCA, MHAW];
-export const programsHCA = [HCA];
-export const programsMHAW = [MHAW];
+export const Program = {
+  HCA: 'HCA',
+  MHAW: 'MHAW',
+  NonHCAP: 'Non-HCAP',
+};
+
+export const allPrograms = [Program.HCA, Program.MHAW];
+export const programsHCA = [Program.HCA];
+export const programsMHAW = [Program.MHAW];
```

Comment:
I made a `constants/programs.js` file a little bit ago with a `Program` const, would adding `NonHCAP` to that work the same (It has `Unknown` as well)? and could we move this and `Program` into there?  just to match the purpose of the file